server/api/region_test.go

```diff
@@ -331,3 +333,72 @@ func (s *testGetRegionSuite) TestScanRegionByKey(c *C) {
 		c.Assert(v, Equals, regions.Regions[i].ID)
 	}
 }
+
+// Create n regions (0..n) of n stores (0..n).
+// Each region contains np peers, the first peer is the leader.
+// (copied from server/cluster_test.go)
+func newTestRegions(n, np uint64) []*core.RegionInfo {
+	regions := make([]*core.RegionInfo, 0, n)
+	for i := uint64(0); i < n; i++ {
+		peers := make([]*metapb.Peer, 0, np)
+		for j := uint64(0); j < np; j++ {
+			peer := &metapb.Peer{
+				Id: i*np + j,
+			}
+			peer.StoreId = (i + j) % n
+			peers = append(peers, peer)
+		}
+		region := &metapb.Region{
+			Id:          i,
+			Peers:       peers,
+			StartKey:    []byte{byte(i)},
```

Comment:
maybe use `fmt.Sprintf("%d",i)`, `i ` bigger than uint8 in your case.

Reply:
Done